src/web/timeline.rkt

```diff
@@ -221,9 +221,10 @@
   (match-define (list (list sizes compileds) ...) compiler)
   (define size (apply + sizes))
   (define compiled (apply + compileds))
+  (define ratio (if (zero? size) 0 (- 1 (/ compiled size))))  ; 0/0 means 0% improvement
   `((dt "Compiler")
     (dd (p "Compiled " ,(~a size) " to " ,(~a compiled) " computations "
-           "(" ,(format-percent (- 1 (/ compiled size))) " saved)"))))
+           "(" ,(format-percent ratio) " saved)"))))
```

Comment:
Good change but how is it possible to have a compiled size of 0? Is that where the body is a variable?